bip-0148.mediawiki

```diff
@@ -24,7 +24,9 @@ This document specifies a BIP16 like soft fork flag day activation of the segreg
 
 Segwit increases the blocksize, fixes transaction malleability, and makes scripting easier to upgrade as well as bringing many other [https://bitcoincore.org/en/2016/01/26/segwit-benefits/ benefits].
 
-It is hoped that miners will respond to this BIP by activating segwit early, before this BIP takes effect. Otherwise this BIP will cause the mandatory activation of the existing segwit deployment before the end of midnight November 15th 2017.
+It is hoped that miners will respond to this BIP by activating segwit early, before this BIP takes effect. Otherwise this BIP may cause the mandatory activation of the existing segwit deployment before the end of midnight November 15th 2017.
```

Comment:
"_**before the end of** midnight_" can simply be written as "_**before** midnight_"  (I know you didn't write this part, but would be nice to update if making changes to this line)